Bindings/Java/Matlab/Dynamic_Walker_Example/DesignMainStarter_Manager.m

```diff
@@ -0,0 +1,75 @@
+function J = DesignMainStarter_Manager(states)
+% This script opens a Model, edits its initial state, runs a forward 
+% simulation.
+
+display(states)
+
+% Import
+import org.opensim.modeling.*;
+
+% Open a Model by name
+model = Model('../Model/WalkerModel.osim');
+
+% Initialize the system and get the initial state
+s = model.initSystem();
+
+% Set the initial states of the model
+editableCoordSet = model.updCoordinateSet();
+editableCoordSet.get(3).setValue(s, states(1)); % LHIP
+editableCoordSet.get(4).setValue(s, states(1)); % RHIP
+editableCoordSet.get(5).setValue(s, states(1)); % LKnee
+editableCoordSet.get(6).setValue(s, states(1)); % RKnee
+
+% add a reporter
+reporter = TableReporter();
+reporter.set_report_time_interval(0.1)
+
+% Add inputs to the reporter
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(1).getOutput('value'), 'Pelvis_tx')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(3).getOutput('value'), 'Hip_Angle_L')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(4).getOutput('value'), 'Hip_Angle_R')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(5).getOutput('value'), 'Knee_Angle_L')
+reporter.updInput('inputs').connect( model.getCoordinateSet().get(6).getOutput('value'), 'Knee_Angle_R')
+
+% add the reporter to the model
+model.addComponent(reporter) 
+
+% re-initialize the state
+state = model.initSystem();
```

Comment:
Should L38 be removed since `state` isn't being used (and the coordinates will have the wrong initial values)? However, it seems like you need to call `initSystem()` after `model.addComponent(reporter);`. Perhaps first create/add the reporter, then call `initSystem()`, and then set the coordinate values? Further, it may be clearer to configure the model first, and set the initial conditions immediately before simulating.